cmd/bclient/bserver/chunkfile.go

```diff
@@ -0,0 +1,58 @@
+package bserver
+
+import (
+	"crypto/md5"
+	"fmt"
+	"io"
+	"os"
+	"path"
+)
+
+const BogusFileId string = "Uninitialized"
+
+func chunkAndUpload(fileroot string, srcFile string, srcFileMd5 []byte, item string, fileChunkSize int) (string, error) {
+
+	sourceFile, err := os.Open(path.Join(fileroot, srcFile))
+
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+
+	defer sourceFile.Close()
+
+	chunk := make([]byte, fileChunkSize)
+
+	var fileId string = BogusFileId
```

Comment:
Why not just use the empty string `""` to indicate whether the fileId has been assigned yet?